lib/utils/beforeBlockString.js

```diff
@@ -1,11 +1,20 @@
 /* @flow */
 'use strict';
 
-module.exports = function(statement /*: Object*/, options /*:: ?: Object*/) /*: string*/ {
-	options = options || {};
-
+/** @typedef {import('postcss').Rule} Rule */
+/** @typedef {import('postcss').AtRule} AtRule */
+
+/**
+ * @param {Rule | AtRule} statement
+ * @param {{
+ * 	noRawBefore?: boolean
+ * }} options
```

Comment:
Should be `@returns` type here? Sometimes it's present and sometimes not :)
It's also absent in some files:
`lib/utils/blurComments.js`
`lib/utils/blurInterpolation.js`
`lib/utils/containsString.js`
`lib/utils/declarationValueIndex.js` for example.

Reply:
Nice catch!

Reply:
I was adding missing `@returns` and then I remembered that we don't need to specify all types always, because TypeScript could infer them from code. So I didn't look through all files. Even types that I added in the last commit (https://github.com/stylelint/stylelint/pull/4387/commits/a41fbcb447d62bf8c169c51a8fcc4d8893cba9c4) are probably unnecessary. @vankop correct me if I'm wrong.

Reply:
I think you're right 👍